build.yaml

```diff
@@ -102,7 +102,24 @@ release:
                   find build -type f -name "*.${PACKAGE_TYPE}" -exec mv {} packages \; 
                   rm -rf build
                   PHP_TOKENS=($(echo ${PHP_VERSION} | tr "-" " "))
-                  sudo ${PACKAGE_UTILITY_REMOVE} ${PHP_TOKENS[0]}-common
+                  # Installing packaging and testing connection
+                  sudo ${PACKAGE_INSTALL} packages/${PHP_TOKENS[0]}*.${PACKAGE_TYPE}
+                  php <<EOF
+                  <?php
+                  \$cluster = Cassandra::cluster()->build();
+                  \$keyspace = 'system';
+                  try{
+                    \$session = \$cluster->connect(\$keyspace);
+                  } catch (Cassandra\\Exception\\RuntimeException \$e) {
+                    echo 'Caught exception: ',  \$e->getMessage(), "\n";
+                  }     
+            EOF
+                  # Throw error if connection test fails
+                  if [ $? -ne 0 ] ; then
```

Comment:
I'd would like this to check the output of the test script above. It's sort of paranoid, but I want to prove that the actual code ran and exit codes can't fully do that because it's more likely that `php` falsely returns a 0 exit code than randomly printing "Success".